packages/fondue/src/components/RichTextEditor/Plugins/CheckboxListPlugin/CheckboxListElement.tsx

```diff
@@ -50,7 +49,10 @@ export const CheckboxListElementNode = (props: PlateRenderElementProps) => {
 };
 
 export class CheckboxListElement extends MarkupElement {
-    constructor(id = ELEMENT_CHECK_ITEM, node = CheckboxListElementNode) {
+    constructor(
+        id = ELEMENT_CHECK_ITEM,
+        node: (props: PlateRenderElementProps & { style: CSSProperties }) => JSX.Element = CheckboxListElementNode,
```

Comment:
Maybe not required to type this as it's in the same file as the typed function

Reply:
It was actually needed, without that the build fails.